src/components/Dashboard/ProgressBar/ProgressBar.js

```diff
@@ -1,15 +1,45 @@
-import React from "react";
+import React, { Component } from "react";
 import styles from "./ProgressBar.scss";
 
-const ProgressBar = () => {
-  return (
-    <ul className={styles.ProgressBar}>
-      <li>Dane dziecka</li>
-      <li>Wnioskodawcy</li>
-      <li>Zespół</li>
-      <li>Podsumowanie</li>
-    </ul>
-  );
-};
+class ProgressBar extends Component {
+  render() {
+    let menuStep1;
+    let menuStep2;
+    let menuStep3;
+    let menuStep4;
+
+    // eslint-disable-next-line react/prop-types
```

Comment:
You can add actual proptypes, how to add them:
https://github.com/facebook/prop-types